optimizer_scripts/tools/combo.py

```diff
@@ -130,15 +130,17 @@ def pytorch_constant_folding(m):
     replacing.replace_shape_with_constant(m.graph)
 
     # constant_folding
+    m = other.inference_shapes(m)
```

Comment:
Why here are two inference shapes? Can we combine them?

Reply:
for the first costant_folding:
sometimes we need the shape information to start constant_folding. and maybe other.inference_shapes could help to calculate it.

for the second constant_folding:
some shape could be calculated only after constant_folding, that's why we need other.inference_shapes here.